packages/portal/spec-renderer/src/components/SpecOperationsList.vue

```diff
@@ -180,6 +199,7 @@ const uid = computed<string>(() => uuidv1())
 
 const filterQuery = ref<string>('')
 const taggedItems = ref<OperationListItem[]>([])
+const untaggedItemsSectionName = ref('default')
```

Comment:
question: If this doesn't change from `default` does it need to be a `ref`?

Reply:
Moved to `const.ts`